doc/cpd/101.rst

```diff
@@ -75,6 +78,16 @@ if set to ``True`` would use the secrets handler.
 The secrets handler would be automatically added to the ``StorageHandlerManager`` and
 would require no special configuration by the operator.
 
+New HTTP Handler
+~~~~~~~~~~~~~~~~
+A new handler called ``secrets`` could be added. This would proxy requests back to the
+Custodia instance.
```

Comment:
One other piece to this is how `commissaire-storage-service` learns where the `/api/v0/secrets` endpoint is.  Might be nice if `commissaire-http` could slip this info to it somehow so we can avoid another line item in `storage.conf`.